tests/aws/services/logs/test_logs.py

```diff
@@ -289,9 +287,9 @@ def test_put_subscription_filter_lambda(
         result = aws_client.lambda_.add_permission(
             FunctionName=test_lambda_name,
             StatementId=test_lambda_name,
-            Principal=f"logs.{config.AWS_REGION_US_EAST_1}.amazonaws.com",
+            Principal=f"logs.{TEST_AWS_REGION_NAME}.amazonaws.com",
             Action="lambda:InvokeFunction",
-            SourceArn=f"arn:aws:logs:{config.AWS_REGION_US_EAST_1}:{account_id}:log-group:{logs_log_group}:*",
+            SourceArn=f"arn:aws:logs:{TEST_AWS_REGION_NAME}:{account_id}:log-group:{logs_log_group}:*",
```

Comment:
Assertions expected the `us-east-1` region even when the resources were created in a different region (by setting `TEST_AWS_REGION_NAME`)